earth_enterprise/src/common/fileaccessor_unittest.cpp

```diff
@@ -0,0 +1,22 @@
+#include <gtest/gtest.h>
+#include "FileAccessor.h"
+
+using namespace std;
+using namespace testing;
+
+TEST(FileAccessorTest, validate_lines_from_file) {
+    string filename = "/tmp/TestFile.txt";
+    vector<string> lines;
+    unique_ptr<AbstractFileAccessor> aFA = AbstractFileAccessor::getAccessor(filename);
+
+    aFA->Open(filename, "w");
+    assert (aFA->getFD() != -1);
+    aFA->Close();
```

Comment:
It looks like this test is simply opening and closing the file. Is it supposed to be calling `GetLinesFromFile`?